src/test/resources/io/jenkins/plugins/pipelinegraphview/utils/gh233_singleRunningMultipleNestedParallelBranch.jenkinsfile

```diff
@@ -20,9 +20,9 @@ pipeline {
                             stage("Test B") {
                                 parallel([
                                     "B1" : {
-                                       sleep(1)
-                                       echo("Starting sleep B1...")
-                                       sleep(1)
+                                        // sleep to make it more reliable the other branch has finished
+                                        sleep '1'
```

Comment:
I didn't manage to get rid of all the sleeps here as often the other branch doesn't finish in time.